src/plugins/saved_objects_finder/public/finder/saved_object_finder.tsx

```diff
@@ -307,13 +314,23 @@ export class SavedObjectFinderUi extends React.Component<
           );
 
           const tooltipText = this.props.getTooltipText?.(item);
-
+          const description = descriptionsDefined && (
```

Comment:
This could be

```ts
const description = !!item.simple.attributes.description && (
  <EuiText size="xs" color="subdued" className="eui-textTruncate">
    {item.simple.attributes.description}
  </EuiText>
)
```

Reply:
ooooo TIL boolean !!